### ISSUE TITLE
Error message "500: Cannot change phone number" when registering

### ISSUE THREAD

**Issue by emanuelen5** [makerspace/makeradmin#584]
Recently a friend of mine signed up for Stockholm Makerspace and got an error in the sign-up flow:

"500: Cannot change phone number" (I don't remember the exact wording)

Other than that, the sign-up flow worked. But it is confusing for potential members to get that type of error message.

<hr/>

Device: Phone
Browser: Firefox

**Comment by BerglundDaniel** [comment]
Is this maybe related to #468?

**Comment by emanuelen5** [comment]
No, it was a completely new email address that was used on this sign-up. But, both relate to the sign up process.
